src/test/storage/storage_manager_test.cpp

```diff
@@ -8,46 +8,62 @@
 
 namespace opossum {
 
-// class StorageStorageManagerTest : public BaseTest {
-//  protected:
-//   void SetUp() override {
-//     auto& sm = StorageManager::get();
-//     auto t1 = std::make_shared<Table>();
-//     auto t2 = std::make_shared<Table>(4);
-
-//     sm.add_table("first_table", t1);
-//     sm.add_table("second_table", t2);
-//   }
-// };
-
-// TEST_F(StorageStorageManagerTest, GetTable) {
-//   auto& sm = StorageManager::get();
-//   auto t3 = sm.get_table("first_table");
-//   auto t4 = sm.get_table("second_table");
-//   EXPECT_THROW(sm.get_table("third_table"), std::exception);
-// }
-
-// TEST_F(StorageStorageManagerTest, DropTable) {
-//   auto& sm = StorageManager::get();
-//   sm.drop_table("first_table");
-//   EXPECT_THROW(sm.get_table("first_table"), std::exception);
-//   EXPECT_THROW(sm.drop_table("first_table"), std::exception);
-// }
-
-// TEST_F(StorageStorageManagerTest, ResetTable) {
-//   StorageManager::get().reset();
-//   auto& sm = StorageManager::get();
-//   EXPECT_THROW(sm.get_table("first_table"), std::exception);
-// }
-
-// TEST_F(StorageStorageManagerTest, DoesNotHaveTable) {
-//   auto& sm = StorageManager::get();
-//   EXPECT_EQ(sm.has_table("third_table"), false);
-// }
-
-// TEST_F(StorageStorageManagerTest, HasTable) {
-//   auto& sm = StorageManager::get();
-//   EXPECT_EQ(sm.has_table("first_table"), true);
-// }
+class StorageStorageManagerTest : public BaseTest {
+ protected:
+  void SetUp() override {
+    auto& sm = StorageManager::get();
+    auto t1 = std::make_shared<Table>();
+    auto t2 = std::make_shared<Table>(4);
+
+    sm.add_table("first_table", t1);
+    sm.add_table("second_table", t2);
+  }
+};
+
+TEST_F(StorageStorageManagerTest, GetTable) {
+  auto& sm = StorageManager::get();
+  auto t3 = sm.get_table("first_table");
+  auto t4 = sm.get_table("second_table");
+  EXPECT_THROW(sm.get_table("third_table"), std::exception);
```

Comment:
Right now you are just checking if something is received, but not if the right thing comes back. A check if the return value is the right table would be nice.